src/Symfony/Component/Messenger/Envelope.php

```diff
@@ -127,6 +127,6 @@ private function resolveAlias(string $fqcn): string
     {
         static $resolved;
 
-        return $resolved[$fqcn] ?? ($resolved[$fqcn] = (new \ReflectionClass($fqcn))->getName());
+        return $resolved[$fqcn] ?? ($resolved[$fqcn] = class_exists($fqcn) ? (new \ReflectionClass($fqcn))->getName() : $fqcn);
```

Comment:
isn't `$fqcn` === `(new \ReflectionClass($fqcn))->getName()` ?

Reply:
Not if `$fcqn` is a class alias.